src/driver/mysql/MysqlDriver.ts

```diff
@@ -154,6 +154,19 @@ export class MysqlDriver implements Driver {
         }, this.options.extra || {});
 
         this.pool = this.mysql.createPool(options);
+        
+        return new Promise<void>((ok, fail) => {
+            this.pool.getConnection((err: any, connection: any) => {
```

Comment:
you need to call `connection.release()` because you need to release each connection you are getting from pool